docs/source/appendices/development_process.rst

```diff
@@ -70,7 +70,7 @@ version, a request for community review will be indicated by a beta
 release of the new version. Community stakeholders involved in the
 feature requests and requirements gathering for the included features
 are notified by Project Maintainers for review and approval of the
-release. After a community review period of at least two weeks, the
+release. After a community review period of at least one week, the
```

Comment:
I think we want to keep things moving, but I think one week is a very short period for community review unless the proposed changes are limited in scope and relatively straightforward. The outcomes of this change (from 2 weeks to 1 week) could include discouraging feedback and an increase in discussion of changes that have already been merged. I would prefer to keep the original time frame of 2 weeks.

Reply:
One week is ample, IMO. Larry, Alex, and I all concurred on this point.